integration/integration_test.go

```diff
@@ -1089,7 +1089,7 @@ func (s *IntSuite) TestTwoClustersTunnel(c *check.C) {
 
 		// wait for both sites to see each other via their reverse tunnels (for up to 10 seconds)
 		abortTime := time.Now().Add(time.Second * 10)
-		for len(b.Tunnel.GetSites()) < 2 && len(b.Tunnel.GetSites()) < 2 {
```

Comment:
Man... I've fixed this issue a few times already I feel. Either it keeps getting reverted somehow or it has always lived in multiple places.